adafruit_wiznet5k/adafruit_wiznet5k.py

```diff
@@ -175,6 +175,14 @@ def __init__(
         self.mac_address = mac
         self._src_port = 0
         self._dns = 0
+
+        # First wait link status is on
+        # to avoid the code during DHCP - assert self.link_status, "Ethernet cable disconnected!"
+        while True:
```

Comment:
Someone else may have another opinion, but if this never is successful the user will never know it will loop forever and there is no timeout.

Reply:
how about 6473abe8a3dd1a986514d54e4d6afb7aefdb1f52 ?